pkg/arm/v7/data/master-startup.sh

```diff
@@ -169,11 +169,10 @@ systemctl stop rsyslog.service
 
 # note: atomic-openshift-node crash loops until master is up
 systemctl enable atomic-openshift-node.service
-systemctl start atomic-openshift-node.service &
 {{ if .Config.SecurityPatchPackages }}
-needs-restarting --reboothint &>/dev/null || {
-  logger -t master-startup.sh "rebooting $(hostname) to complete ARO security updates"
-  shutdown --reboot now
-}
+logger -t master-startup.sh "scheduling $(hostname) reboot to complete ARO security updates"
+shutdown --reboot +5
 {{end}}
+systemctl start atomic-openshift-node.service &
```

Comment:
I know this is an artifact but what is the intention of the `&` at the end here?  Why do we want to background the start of openshift-node?

Reply:
+1, systemctl will handle the daemonization of this process. Not sure why we need to fork `systemctl` into the background.

Reply:
it's a hack - if a node tries to start before the masters are up (likely), the kubelet will `exit 1` after some time.  systemd will keep trying to start the kubelet, but the initial kubelet `exit 1` causes `systemctl start` to `exit 1`, and hence the script to `exit 1`.

Could do `|| true` here instead if preferred.

Reply:
I'm fine either way but adding a note might be helpful for future.  Thanks @jim-minter

Reply:
I think the intent is clearer if we use `|| true` so I'd prefer that.